pagopa/io-app
# chore: [IA-536] Add remote zendesk config

## THIS PR DEPENDS ON https://github.com/pagopa/io-services-metadata/pull/526

## Short description
This PR add the client, the action and the reducer needed to manage the remote zendesk configuration.

## List of changes proposed in this pull request
- Added the client
- Added the saga that manage the API call
- Added the action and the reducer
- Dispatched the action when the Zendesk workflow starts


https://user-images.githubusercontent.com/11773070/145378302-08e4935d-f9bb-4070-ad59-36bad034b5d1.mov



## How to test
Make sure to checkout this branch on dev-server: https://github.com/pagopa/io-dev-api-server/pull/103
Set the `zendeskEnabled` FF to `true`
Start the ask assistance flow pressing "**?**"
